pkg/controller/servicebinding/reconciler_test.go

```diff
@@ -391,10 +407,18 @@ func TestReconcilerUpdateCredentials(t *testing.T) {
 	r := &reconciler{dynClient: fakeDynClient, restMapper: mapper, scheme: f.S}
 	r.resourceWatcher = newFakeResourceWatcher(mapper)
 
-	u, err := fakeDynClient.Resource(secretsGVR).Get("db-credentials", metav1.GetOptions{})
+	res, err := r.Reconcile(reconcileRequest())
```

Comment:
I'm unsure if calling `Reconcile` twice is what we want in this test. Perhaps we're testing two distinct scenarios instead? Could you elaborate why this is required?